Assets/Javascript/index2.js

```diff
@@ -0,0 +1,251 @@
+    elmArray.push([detFeatures, detdot2, 'det_feature_dot']);
+    elmArray.push([detFeatures, detFeat3, 'det_feature det_f_1', workInformation[i].features[2]]);
+    elmArray.push([detailBackground, detImage, 'detail_img']);
+    elmArray.push([detailBackground, detDescription, 'detail_descr', workInformation[i].detailText]);
+    elmArray.push([detailBackground, languagesDiv, 'language_box']);
+    elmArray.push([languagesDiv, languagesDivLang1, 'language_prop', workInformation[i].languages[0]]);
+    elmArray.push([languagesDiv, languagesDivLang2, 'language_prop', workInformation[i].languages[1]]);
+    elmArray.push([languagesDiv, languagesDivLang3, 'language_prop', workInformation[i].languages[2]]);
+    elmArray.push([detailBackground, buttonsDiv, 'det_buttons_div']);
+    elmArray.push([buttonsDiv, buttonsDivBtn1, 'det_button']);
+    elmArray.push([buttonsDiv, buttonsDivBtn2, 'det_button']);
+    elmArray.push([buttonsDivBtn1, button1Content1, 'det_button_text', 'See live']);
+    elmArray.push([buttonsDivBtn1, button1Content2, 'det_button_icon']);
+    elmArray.push([buttonsDivBtn2, button2Content1, 'det_button_text', 'See Source']);
+    elmArray.push([buttonsDivBtn2, button2Content2, 'det_button_icon']);
+    constructor(elmArray);
+    button1Content2.src = './Assets/Images/detail_btn1.svg';
+    button2Content2.src = './Assets/Images/detail_btn2.svg';
+    detCloseBtn.src = './Assets/Images/x2.svg';
+    detdot1.src = './Assets/Images/dot.png';
+    detdot2.src = './Assets/Images/dot.png';
+    detdot1.alt = 'dot';
+    detdot2.alt = 'dot';
+    button1Content2.alt = 'See live icon';
+    button2Content2.alt = 'See source';
+    detCloseBtn.alt = 'Close';
+    detailBackground.style = 'animation-name: detail_open; animation-duration: 0.5s;';
+    detailPreBackground.style = 'animation-name: background-open; animation-duration: 0.5s;';
+
+    for (let i = 0; i < 5; i += 1) {
+      sectObj[i].style = 'filter: blur(4px); -webkit-filter: blur(4px);';
+    }
+    detCloseBtn.addEventListener('click', (event) => {
+      event.preventDefault();
+      elmArray = [];
+      detailBackground.style = 'animation-name: detail_close;animation-duration: 1s;';
+      detailPreBackground.style = 'animation-name: background-close; animation-duration: 1s;';
+      setTimeout(() => { detailPreBackground.remove(); detailBackground.remove(); }, 800);
+      for (let i = 0; i < 5; i += 1) {
+        sectObj[i].style = '';
+      }
+    });
+    detailPreBackground.addEventListener('click', (event) => {
+      event.preventDefault();
+      elmArray = [];
+      detailBackground.style = 'animation-name: detail_close;animation-duration: 1s;';
+      detailPreBackground.style = 'animation-name: background-close; animation-duration: 1s;';
+      setTimeout(() => { detailPreBackground.remove(); detailBackground.remove(); }, 800);
+      for (let i = 0; i < 5; i += 1) {
+        sectObj[i].style = '';
+      }
+    });
+  });
+}
+constructor(ElementsArray);
```

Comment:
I suggest you should only render the projects when the page is loaded. Meaning that all the HTML of the project section should be constructed when the page loads. One way of implementing that is: 
```JavaScript
window.onload = () => {
  // The DOM elements creation or functions that create them are called here. e.g.
  constructor(ElementsArray);
};

```

Reply:
Hello @eri8-9 ! Please note that the `window.onload` is an optional suggestion. We have discussed that with @teekaytech :) We are sorry for misleading you, our intention was to make your portfolio app better :)

Reply:
> Hello @eri8-9 ! Please note that the `window.onload` is an optional suggestion. We have discussed that with @teekaytech :) We are sorry for misleading you, our intention was to make your portfolio app better :)

Hello @nidalaa, you tag the wrong person. This is not my project. Please don't do that.